### PR TITLE
Change links from acx_api api to acx_gallery where necessary

### LINKED ISSUE #1771: ACX codelab: feature ACX gallery earlier
In https://github.com/dart-lang/site-webdev/pull/1769#pullrequestreview-144597193, @kwalrath wrote:

> I think (in a later PR) we should make the gallery a bigger part of this codelab. We could feature the gallery in the first page, and we could refer to it instead of the API docs when talking about a component.

### PR DESCRIPTION
**Pull request by artem-galas**
Fixes #1771

### CODE REVIEW

**Comment by kwalrath** [comment]
Thanks for the PR, @artem-galas! I'm going to stage this so we can verify the links.

File: src/_codelabs/angular_components/2-starteasy.md [hunk 1]
```diff
@@ -312,7 +312,7 @@ Adding those two lines to `lib/src/help/help.dart` makes the material icons disp
 ## <i class="far fa-money-bill-alt fa-sm"> </i> Use acx-scorecard
 
 Make one more change: use
-[ScorecardComponent]({{site.acx_api}}/angular_components/ScorecardComponent-class.html)
+[ScorecardComponent]({{site.acx_gallery}}/#/scorecard)
```
[the remark this answers is not in the document]
Comment (artem-galas) [reply src/_codelabs/angular_components/2-starteasy.md]: Actually I had the same thoughts :)
I'll update the brunch or create a new one and leave pr?

[the remark this answers is not in the document]
Comment (kwalrath) [reply src/_codelabs/angular_components/2-starteasy.md]: New branch is probably easiest, unless you have good git-fu.

[the remark this answers is not in the document]
Comment (artem-galas) [reply src/_codelabs/angular_components/2-starteasy.md]: updated here #1871

**Pull request merged**